holoviews/core/data/interface.py

```diff
@@ -359,7 +359,17 @@ def dtype(cls, dataset, dimension):
         else:
             return data.dtype
 
-
+    @classmethod
+    def replace_value(cls, data, nodata):
+        """
+        Replace `nodata` value in data with NaN
+        """
+        data = data.astype('float64')
+        mask = data != nodata
+        if hasattr(data, 'where'):
+            return data.where(mask, np.NaN)
+        return np.where(mask, data, np.NaN)
+        
```

Comment:
I guess this duplicates `apply_nodata._replace_value` in `plotting.util` a bit but that is probably ok...